supertokens_python/recipe/session/with_jwt/recipe_implementation.py

```diff
@@ -189,12 +188,16 @@ async def update_access_token_payload(
         new_access_token_payload: Dict[str, Any],
         user_context: Dict[str, Any],
     ) -> bool:
-        # TODO: Node SDK allows non dict values of new_access_token_payload. But what should we do here?
+        deprecated_warn(
+            "update_access_token_payload is deprecated. Use merge_into_access_token_payload instead"
+        )
```

Comment:
remove this from here - it should print this only in user facing function for update_access_token_payload

Reply:
done.